app/src/GUI.py

```diff
@@ -379,7 +380,8 @@ def draw(self):
             name.pack(anchor=W, fill=X, expand=True)
             text = TIMESTAMP.format(stu.timestamp)
             if stu.sent_mail:
-                text += ' (Mail Invite Sent)'
+                hour = strftime(DATETIME_FORMAT, localtime())
+                text += f' (Mail Invite Sent {hour})'
```

Comment:
I'm not certain this will work, as I am pretty sure this will be called each time the list is populated, so the mail timestamp will be constantly updating.
I think the only correct way to do this is by putting a timestamp in the google sheet every time an email is sent. 
currently a marker is written in a special column to note that a mail has been sent, so that marker can be replaced by a timestamp that can then be read when updating the list.

Reply:
Just to clarify, there already is a column in the sheet used to mark that a mail was sent, so it will be relatively simple to change the format of this mark from what it is now (something like "SENT") to a timestamp.